azurerm/resource_arm_notification_hub_namespace.go

```diff
@@ -176,9 +200,21 @@ func resourceArmNotificationHubNamespaceRead(d *schema.ResourceData, meta interf
 		d.Set("location", azureRMNormalizeLocation(*location))
 	}
 
-	sku := flattenNotificationHubNamespacesSku(resp.Sku)
-	if err := d.Set("sku", sku); err != nil {
-		return fmt.Errorf("Error setting `sku`: %+v", err)
+	if skuName == "" {
```

Comment:
As above, we can mark them as computed and set both

Reply:
Not sure what that will do to the statefile or compair... I will make the logic easier and remove unneeded functions...